tests/test_enterprise/api/test_views.py

```diff
@@ -9976,6 +9976,52 @@ def test_list_users_filtered(self):
 
 @ddt.ddt
 @mark.django_db
+class TestEnterpriseCustomerMembers(BaseTestEnterpriseAPIViews):
+    """
+    Test enterprise customer members list endpoint
+    """
+    ECM_ENDPOINT = 'enterprise-customer-members'
+    ECM_KWARG = 'enterprise_uuid'
+
+    def test_get_enterprise_org_members(self):
+        """
+        Assert whether the response is valid.
+        """
+        user_1 = factories.UserFactory(first_name="Rhaenyra")
+        user_2 = factories.UserFactory(first_name="Jace")
+        user_3 = factories.UserFactory(first_name="Alicent")
+
+        enterprise_customer = factories.EnterpriseCustomerFactory(uuid=FAKE_UUIDS[0])
+        factories.EnterpriseCustomerUserFactory(
+            user_id=user_1.id,
+            enterprise_customer=enterprise_customer
+        )
+        factories.EnterpriseCustomerUserFactory(
+            user_id=user_2.id,
+            enterprise_customer=enterprise_customer
+        )
+        factories.EnterpriseCustomerUserFactory(
+            user_id=user_3.id,
+            enterprise_customer=enterprise_customer
+        )
+
+        # Test invalid UUID
+        url = reverse(self.ECM_ENDPOINT, kwargs={self.ECM_KWARG: 123})
+        response = self.client.get(settings.TEST_SERVER + url)
+        self.assertEqual(response.status_code, 404)
+
+        # Test valid UUID
+        url = reverse(self.ECM_ENDPOINT, kwargs={self.ECM_KWARG: enterprise_customer.uuid})
+        response = self.client.get(settings.TEST_SERVER + url)
+        data = response.json().get('results')
+
+        # list should be sorted alphabetically by name
+        print(data)
+        self.assertEqual(data[0]['enterprise_customer_user']['first_name'], user_3.first_name)
+        self.assertEqual(data[1]['enterprise_customer_user']['first_name'], user_2.first_name)
+        self.assertEqual(data[2]['enterprise_customer_user']['first_name'], user_1.first_name)
+
+
 class TestDefaultEnterpriseEnrollmentIntentionViewSet(BaseTestEnterpriseAPIViews):
```

Comment:
I think the test is failing because the decorators `@ddt.ddt` and `@mark.django_db` were removed when `TestEnterpriseCustomerMembers` was added. When adding them back in, note this viewset's tests show 17 tests on `master` vs. 16 tests on this PR. If ddt was properly working for the test class, the `test_default_enrollment_intentions_learner_status_already_enrolled_active_audit` would actually be two tests:

* `test_default_enrollment_intentions_learner_status_already_enrolled_active_audit_1`
* `test_default_enrollment_intentions_learner_status_already_enrolled_active_audit_2`